gedcom/__init__.py

```diff
@@ -917,22 +917,34 @@ def get_name(self):
         last = ""
         if not self.is_individual():
             return first, last
+        
+        # Return the first GEDCOM_TAG_NAME that is found.  Alternatively
+        # as soon as we have both the GETCOM_TAG_GIVEN_NAME and _SURNAME return those
+        found_given = False
```

Comment:
@nomadyow May you rename it to `found_given_name`? It's more descriptive I think :)

Reply:
Done!